docs/_advanced/token_rotation.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Token rotation
+lang: en
+slug: token-rotation
+order: 3
+---
+
+<div class="section-content">
+Supported in Bolt for JavaScript as of v3.5.0, token rotation provides an extra layer of security for your access tokens and is defined by the [OAuth V2 RFC](https://datatracker.ietf.org/doc/html/rfc6749#section-10.4). 
+
+Instead of an access token representing an existing installation of your Slack app indefinitely, with token rotation enabled, access tokens expire. A refresh token acts as a long-lived way to refresh your access tokens.
+
+Bolt for JavaScript supports token rotation automatically once enabled in your app's configuration.
```

Comment:
As one of the prerequisites on the Bolt app side, we can mention the necessity to use `@slack/oauth` package's default `authorize` function along with `InstallationStore`. This sounds obvious to us but it's not the same particularly for the developers who are new to Bolt. 

Also, we may want to update [the OAuth package's document](https://slack.dev/node-slack-sdk/oauth) as well.